tests/api_validation_test.go

```diff
@@ -0,0 +1,160 @@
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/ghodss/yaml"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	dataVolumeName     = "test-dv"
+	validURL           = "http://www.example.com/example.img"
```

Comment:
If you are going to change the code per my comment below about other source types, then please change validURL to validURLFormat too. If you are not, then that's ok. It's not critical of course